### ISSUE TITLE
Draggable message windows

### ISSUE THREAD

**Issue by sadettinpolat** [sweetalert2/sweetalert2#2673]
### New feature motivation

The dragging feature is useful to view the information behind the message window.

### New feature description

```
Swal.fire({
  imageUrl: 'https://placeholder.pics/svg/300x1500',
  imageHeight: 1500,
  imageAlt: 'A tall image',
 **draggable:true**
})
```

**Comment by limonte** [comment]
Thank you @sadettinpolat for the feature-request. I'll take a closer look when there are enough reactions from other people.

**Comment by adamfk** [comment]
Being able to move a dialog would be really helpful for a [state machine simulator](https://github.com/StateSmith/StateSmith) I'm working on. Any tips on how users might achieve it without support baked into sweet alert?

Awesome project!

**Comment by plouf142** [comment]
> Thank you @sadettinpolat for the feature-request. I'll take a closer look when there are enough reactions from other people.

@limonte  35 reactions!

**Comment by limonte** [comment]
@plouf142 time to make it 🚀

**Comment by limonte** [comment]
Thank you everyone, implementing this feature probably took longer than it should because it slipped away from my attention (thanks @plouf142 for the ping 😄)

The initial implementation is shipped, please test it out `Swal.fire({ draggable: true })` and leave your feedback here 🙌

**Comment by limonte** [comment]
@plouf142 cursor should be working in the latest (11.15.1)

**Comment by plouf142** [comment]
... 

My bad. 

I'm written "dragging" and so you manage only the grabbing events, and not the grab events, ie the area where the user **_can_** click to enable the dragging! 

Explanations with pictures: 
### Now
**User can't move the SWAL here**
![image](https://github.com/user-attachments/assets/0a1b2f8c-9f2b-4a57-a4d1-297f63e67ea5)

**User can move the SWAL here**
![image](https://github.com/user-attachments/assets/525904aa-f339-46e9-a113-02641add5ce2)


### My requirements
**User can't move the SWAL here**
![image](https://github.com/user-attachments/assets/c1232028-7dfd-4327-9626-c6d729b9ec55)

**User can move the SWAL here**
![image](https://github.com/user-attachments/assets/ad28f8b3-e799-41d8-8567-ced3538b2957)

**Comment by limonte** [comment]
![CleanShot 2024-12-16 at 19 31 54@2x](https://github.com/user-attachments/assets/32ed4a52-30c1-420d-990f-5cb18550aaf7)

@plouf142  isn't it exactly how it works now?

**Comment by plouf142** [comment]
> <img alt="" width="400" src="https://private-user-images.githubusercontent.com/6059356/396205911-32ed4a52-30c1-420d-990f-5cb18550aaf7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ0MjkxNzQsIm5iZiI6MTczNDQyODg3NCwicGF0aCI6Ii82MDU5MzU2LzM5NjIwNTkxMS0zMmVkNGE1Mi0zMGMxLTQyMGQtOTkwZi01Y2IxODU1MGFhZjcucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxNyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTdUMDk0NzU0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzNkZjM0ODBjODAxYmVlZmQyZjU2YmM2ZWI4NTQ4ODg3MzFjY2I4OWNkZmM4YmUxOGVjNzg1M2E3NzQ4YjNhMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.W5QLOvS_NFPjCqlaJ2GyQUh1EIzvEgrufDCE17xwIuo">
> @plouf142 isn't it exactly how it works now?

Hum, no. 

We have "grabbing" cursor between **up and down** events, during the execution of the drag function!

My request is to have "grab" cursor on cursor **hover** area, to tell the user that the drag function is available, that can be enabled at this area.  I will keep the css code https://github.com/sweetalert2/sweetalert2/issues/2673#issuecomment-2546039993 in order to have this behavior.

**Comment by limonte** [comment]
oh, got you, thanks, it's about cursor only.

**Comment by limonte** [comment]
should be good in latest release 👍

**Comment by plouf142** [comment]
Thanks! (Tested, it's ok)

**Comment by limonte** [comment]
Thank you @plouf142  for your valuable feedback! <3

**Comment by sadettinpolat** [comment]
> 🎉 This issue has been resolved in version 11.15.0 🎉
> 
> The release is available on:
> 
> * [npm package (@latest dist-tag)](https://www.npmjs.com/package/sweetalert2/v/11.15.0)
> * [GitHub release](https://github.com/sweetalert2/sweetalert2/releases/tag/v11.15.0)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

thankyou very much.
